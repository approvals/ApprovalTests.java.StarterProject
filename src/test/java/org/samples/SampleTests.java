package org.samples;

import static org.junit.Assert.assertEquals;

import org.approvaltests.Approvals;
import org.junit.Test;

public class SampleTests
{
  @Test
  public void testNormalJunit()
  {
    assertEquals(5, 5);
  }
  @Test
  public void testWithApprovalTests()
  {
    Approvals.verify("Hello World");
  }
  /**
    *  note: this requires GSON which is currently added in the pom.xml file. 
    *  This is only required if you want to use the VerifyAsJson.
    **/
  @Test
  public void testJson()
  {
    Person hero = new Person("jayne", "cobb", true, 38);
    Approvals.verifyAsJson(hero);
  }
}
