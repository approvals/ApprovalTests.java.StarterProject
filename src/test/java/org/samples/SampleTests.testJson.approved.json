{
  "firstName": "jayne",
  "lastName": "cobb",
  "isMale": true,
  "age": 38
}
